pkg/backup/client.go

```diff
@@ -914,8 +959,9 @@ backupLoop:
 					}
 					break
 				}
-				return errors.Annotatef(err, "failed to connect to store: %d with retry times:%d", storeID, retry)
+				return berrors.ErrFailedToConnect.Wrap(err).GenWithStack("failed to connect to store: %d with retry times:%d: %s", storeID, retry, err.Error())
```

Comment:
is it really necessary to include the `err` in GenWithStack? it is already present in Wrap...

```suggestion
				return berrors.ErrFailedToConnect.Wrap(err).GenWithStack("failed to connect to store: %d with retry times:%d", storeID, retry)
```

Reply:
https://github.com/pingcap/errors/issues/33 (This is only for debugging, and I will remove them then. 🥲)